document/src/main/java/com/yahoo/document/DocumentUpdate.java

```diff
@@ -163,8 +167,18 @@ public DocumentUpdate prune(Document doc) {
      *
      * @return a list of all FieldUpdates in this DocumentUpdate
      */
+    @Deprecated
```

Comment:
Consider adding a `@deprecated` JavaDoc entry mentioning the method to use instead

Reply:
Done